rotkehlchen/tests/conftest.py

```diff
@@ -241,7 +242,19 @@ def vcr_cassette_dir(request: pytest.FixtureRequest) -> str:
             log.debug('VCR setup completed')
         else:
             log.debug('Pulling information for VCR cassettes')
-            os.popen(f'cd {tests_caching_dir} && git pull').read()
+            current_branch = os.popen('git rev-parse --abbrev-ref HEAD').read().rstrip('\n')
+            checkout_proc = Popen(f'cd {tests_caching_dir} && git checkout {current_branch}', shell=True, stdout=PIPE, stderr=PIPE)  # noqa: E501
```

Comment:
Shouldn't it try to first fetch/pull from remote? It could be it has the branch but the local does not